db/migrate/20160808171034_create_grocery_lists.rb

```diff
@@ -0,0 +1,10 @@
+class CreateGroceryLists < ActiveRecord::Migration[5.0]
+  def change
+    create_table :grocery_lists do |t|
+      t.string :name
+      t.string :desc
```

Comment:
Please consider using `null: false` for fields that we validate presence of. This would mean the DB is the last safe guard for valid data if someone inserts data without the Rails models.